doc/psyad/user_guide/conf.py

```diff
@@ -10,15 +10,15 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
+import os
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
```

Comment:
Should this lines be removed?

Reply:
Yes although I've taken the opportunity to add the necessary configuration for the apilinks plugin in case we need to refer to the ref guide from this doc in future.